streams/src/test/java/org/apache/kafka/streams/state/internals/MeteredTimestampedWindowStoreTest.java

```diff
@@ -176,38 +183,34 @@ private void doShouldPassChangelogTopicNameToStateStoreSerde(final String topic)
             keySerde,
             valueSerde
         );
-        store.init((StateStoreContext) context, store);
 
+        store.init((StateStoreContext) context, store);
         store.fetch(KEY, TIMESTAMP);
         store.put(KEY, VALUE_AND_TIMESTAMP, TIMESTAMP);
 
-        verify(keySerializer, valueDeserializer, valueSerializer);
+        verify(innerStoreMock).fetch(any(Bytes.class), eq(TIMESTAMP));
```

Comment:
Since both values passed to the `fetch()` method are known (not mocks), we can easily make this verify call more specific: `verify(innerStoreMock).fetch(KEY_BYTES, TIMESTAMP);`

Reply:
Addressed in the next commit